common/src/main/java/org/apache/uniffle/common/util/RssUtils.java

```diff
@@ -292,4 +296,12 @@ public static Roaring64NavigableMap generateTaskIdBitMap(Roaring64NavigableMap b
     }
     return taskIdBitmap;
   }
+
+  public static List<String> getConfiguredLocalDirs(RssConf conf) {
```

Comment:
Could we have a general way to support conf value from env instead of supporting this one by one?

Reply:
For example: marking the config, whose value could be from env. Like that

```
public static final ConfigOption<Long> SERVER_BUFFER_CAPACITY = ConfigOptions
      .key("rss.server.buffer.capacity")
      .longType()
      .noDefaultValue()
      .allowFromEnv()
      .withDescription("Max memory of buffer manager for shuffle server");
```

Once one config option is allowed from env, it will be initialized from env when creating the `shuffleServerConf`. WDYT?

Reply:
Sounds good.

Reply:
It's OK.